modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+[id="hosted-control-planes-concepts_{context}"]
+== Concepts
+
+Management cluster:: An {product-title} cluster where the HyperShift Operator is deployed and where the control planes for tenant clusters are hosted.
+
+Hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+
+Hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+Hosted control plane infrastructure:: Resources on the management cluster or external cloud provider that are prerequisites to run hosted control plane processes.
```

Comment:
While this can provide some entropy, I feel we can ommit to avoid confusion. We already talk about infrastructure with the management cluster, so can be considered redundant.

Reply:
Not the same thing. The hosting cluster is the management cluster. I.e., the cluster that does the hosting act for the control planes. The hosted cluster is the acted upon, that's the cluster provisioned by the cluster service. Hope that makes sense :)

Reply:
So we can probably add a synonym to the management cluster because management/hosting tend to be used interchangeably.